master/buildbot/changes/gerritchangesource.py

```diff
@@ -29,6 +30,33 @@
 from buildbot.util import httpclientservice
 
 
+def canonicalize_event(event):
```

Comment:
is this function a part of the public API for this module?  if not, please use `_canonicalize_event`

Reply:
Done